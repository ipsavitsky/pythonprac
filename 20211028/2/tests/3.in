print(*list(slide(enumerate([7, 14, 12312, 43, 234, 1, 3421]), 2)))