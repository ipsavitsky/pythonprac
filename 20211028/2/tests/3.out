(0, 7) (1, 14) (1, 14) (2, 12312) (2, 12312) (3, 43) (3, 43) (4, 234) (4, 234) (5, 1) (5, 1) (6, 3421) (6, 3421)
